plutarch-test/plutarch-extra/ExtraSpec.hs

```diff
@@ -0,0 +1 @@
+{-# OPTIONS_GHC -F -pgmF sydtest-discover -optF --no-main #-}
```

Comment:
I've come not to like this TBH. I don't think we should just the discover stuff, but that's for another PR.